packages/core/src/Utils/Analytics/index.ts

```diff
@@ -17,45 +17,53 @@ export const AnalyticsInitializer = async () => {
 
+            const analytics_config_config = {
+                loggedIn: !!client_information,
+                account_type: account_type === 'null' ? 'unlogged' : account_type,
+                app_id: String(getAppId()),
+                device_type: window.innerWidth <= MAX_MOBILE_WIDTH ? 'mobile' : 'desktop',
+                device_language: navigator?.language || 'en-EN',
+                user_language: getLanguage().toLowerCase(),
+                country: await CountryUtils.getCountry(),
+                utm_source: ppc_campaign_cookies?.utm_source,
+                utm_medium: ppc_campaign_cookies?.utm_medium,
+                utm_campaign: ppc_campaign_cookies?.utm_campaign,
+                utm_content: ppc_campaign_cookies?.utm_content,
+                domain: window.location.hostname,
+                url: window.location.href,
+                network_type: navigator.connection?.effectiveType,
+                network_rtt: navigator.connection?.rtt,
+                network_downlink: navigator.connection?.downlink,
+                residence_country,
+            };
             const config = {
                 growthbookKey: flags.marketing_growthbook ? process.env.GROWTHBOOK_CLIENT_KEY : undefined,
                 growthbookDecryptionKey: flags.marketing_growthbook ? process.env.GROWTHBOOK_DECRYPTION_KEY : undefined,
                 rudderstackKey: process.env.RUDDERSTACK_KEY,
                 growthbookOptions: {
                     attributes: {
-                        loggedIn: !!client_information,
-                        account_type: account_type === 'null' ? 'unlogged' : account_type,
-                        app_id: String(getAppId()),
-                        device_type: window.innerWidth <= MAX_MOBILE_WIDTH ? 'mobile' : 'desktop',
-                        device_language: navigator?.language || 'en-EN',
-                        user_language: getLanguage().toLowerCase(),
-                        country: await CountryUtils.getCountry(),
-                        utm_source: ppc_campaign_cookies?.utm_source,
-                        utm_medium: ppc_campaign_cookies?.utm_medium,
-                        utm_campaign: ppc_campaign_cookies?.utm_campaign,
-                        utm_content: ppc_campaign_cookies?.utm_content,
-                        domain: window.location.hostname,
-                        url: window.location.href,
-                        network_type: navigator.connection?.effectiveType,
-                        network_rtt: navigator.connection?.rtt,
-                        network_downlink: navigator.connection?.downlink,
-                        residence_country: client_information?.residence,
+                        ...analytics_config_config,
                     },
                 },
             };
             await Analytics?.initialise(config);
+            Analytics.setAttributes(analytics_config_config);
```

Comment:
What is the use of setAttributes here? we have initialized growthbook with same config  in line 65vand nothing changed in attibutes.